src/components/Hero.js

```diff
@@ -16,20 +16,28 @@ export default function Hero() {
         aria-label="Send me an email"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label

> The `aria-label` attribute is intended for interactive elements only. Use `aria-label` to ensure an accessible name is provided when **none** is visible in the DOM [..]

I'm not that skilled with aria-attributes, but I think if your link has a readable (text) label anyway, you don't need an `aria-label`. But then again, it might not matter.

Reply:
Forgot to take it out 😆, you are right though. I changed to readable text to the same as the aria-label and thus the aria-label is quite pointless.